components/TextControls/src/MDCBaseTextField.m

```diff
@@ -662,6 +662,27 @@ - (MDCTextControlColorViewModel *)textControlColorViewModelForState:
   return colorViewModel;
 }
 
+#pragma mark Accessibility Overrides
+
+- (NSString *)accessibilityLabel {
```

Comment:
Please document this behavior in the header of this file.

Reply:
Done.